### PR TITLE
Lachenlama

### PR DESCRIPTION
**Pull request by lachenlama**
# Description

Tetris : The Game made with pygame. Easy to install and play.

Fixes: #632

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Checklist:

- [x ] My code follows the style guidelines(Clean Code) of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] I have created a helpful and easy to understand `README.md`
- [ ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [ ] My changes generate no new warnings
- [x ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by lachenlama** [comment]
#632  Issue PR

File: Tetris/readme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+* Space bar to skip the shape's path and place it to its final position.
+* Upward arrow to rotate the shape.
+* Downward button to move the shape faster.
+* R to restart the game
+* Q to quit the game
+
+### Do run "pip install requirements.txt" before running the main.py file
+
+## Ingame Images
+* ![Ingame image1](https://drive.google.com/file/d/1PlqBkTrg_UqKGECBgChZ7UZfrdCKmaYy/view?usp=sharing)
```
Comment (kaustubhgupta) [inline Tetris/readme.md:18]: images note rendered

File: Tetris/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Package    Version
```
Comment (kaustubhgupta) [inline Tetris/requirements.txt:1]: just mention the package names with version like: pygame==20.2.3 without top heading and next dashed lines

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Code works fine, just some minor changes

File: Tetris/readme.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Tetris : The Game
```
Comment (antrikshmisri) [inline Tetris/readme.md:1]: Add all the dependencies

**Review by antrikshmisri — APPROVED** [review]

**Review by antrikshmisri — CHANGES_REQUESTED** [review]
Add the dependencies in README.md

**Comment by kaustubhgupta** [comment]
@lachenlama Have you looked at the changes suggested?

**Comment by lachenlama** [comment]
Will look into it now sorry, haven't been very active recently @kaustubhgupta

**Comment by lachenlama** [comment]
Please, do check again. If there is any issue do tell me. I am very new in this tbh but i willing to grind @kaustubhgupta

**Review by kaustubhgupta — COMMENTED** [review]
Plagiarism Strike 1
https://levelup.gitconnected.com/writing-tetris-in-python-2a16bddb5318

**Pull request closed without merging**